Microsoft.Toolkit.Uwp.UI.Controls/Graph/PeoplePicker/PeoplePicker.xaml

```diff
@@ -0,0 +1,102 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.Graph"
+                    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+                    xmlns:interactivity="using:Microsoft.Xaml.Interactivity">
+
+    <Style TargetType="controls:PeoplePicker">
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:PeoplePicker">
+                    <StackPanel>
+                        <StackPanel>
+                            <Grid>
+                                <TextBox Name="tbGraphAccessToken"
+                                         Margin="0"
+                                         DataContext="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}}" />
+                                <TextBox Name="SearchBox"
+                                         Margin="0"
+                                         AutomationProperties.Name="Keywords"
+                                         PlaceholderText="{Binding PersonNotSelectedMessage, RelativeSource={RelativeSource Mode=TemplatedParent}}" />
+                                <ProgressRing Name="Loading"
+                                              Margin="0,0,30,0"
+                                              HorizontalAlignment="Right"
+                                              AutomationProperties.Name="Progress"
+                                              Canvas.ZIndex="10"
+                                              FlowDirection="LeftToRight"
+                                              IsActive="False" />
+                            </Grid>
+                        </StackPanel>
+                        <RelativePanel>
+                            <ListBox Name="SearchResultListBox"
+                                     AutomationProperties.Name="Search Results"
+                                     Canvas.ZIndex="100"
+                                     ItemsSource="{Binding SearchResultList, RelativeSource={RelativeSource Mode=TemplatedParent}}"
+                                     Visibility="Collapsed">
+                                <ListBox.ItemTemplate>
+                                    <DataTemplate>
+                                        <!--<TextBlock Text="{Binding UserPrincipalName}" />-->
```

Comment:
Remove commented code.

Reply:
@azchohfi Thanks! We have check and removed the commented code.